.travis.yml

```diff
@@ -1,143 +0,0 @@
-language: python
```

Comment:
this can't be fully deleted yet because GH actions doesn't yet do automated deployments to PyPI and Docker Hub.

I think that we should get the test suites all in and then run GH Actions in parallel with Travis CI, until we have confidence in GH actions and find all the bugs.

This will also give you time to move deployments over and perhaps backport to our active release branches.

Reply:
I've just taken it out for now so I can test gh-actions without running Travis